src/components/TripInfo.tsx

```diff
@@ -0,0 +1,126 @@
+import { useState } from "react";
+import TextareaAutosize from "@mui/base/TextareaAutosize";
+import {
+  TextField,
+  Select,
+  MenuItem,
+  FormControl,
+  InputLabel,
+  Typography,
+} from "@mui/material";
+import style from "../components/style//TripInfo.module.css";
+
+function TripInfo(props: any) {
+  const [form, setForm] = useState({});
+
+  let tripInfoData = {};
+
+  function handlerForm(e: any, key: string) {
+    setForm({
+      ...form,
+      [key]: e.target.value,
+    });
+  }
+
+  tripInfoData = {
+    ...form,
+  };
+  props.onSaveTripInfoData(tripInfoData);
+
+  return (
+    <div className={style.tripInfoContainer}>
+      <Typography className={style.h1}> Trip Info</Typography>
+      <div className={style.acomodation}>
+        <TextField
+          onChange={(e: React.ChangeEvent<HTMLInputElement>) =>
+            handlerForm(e, "accomodation")
+          }
+          className={style.acomodationContainer}
+          id="outlined-basic"
+          label="Type of Accomodation"
+          variant="outlined"
+          color="success"
+        />
+      </div>
+      <FormControl className={style.breakfast}>
+        <InputLabel color="success">Number of Breakfast</InputLabel>
+        <Select
+          onChange={(e: any) => handlerForm(e, "breakfast")}
```

Comment:
Can you try to remove any and try to find out what the type of this event is? Also, the same comment about extracting this function into a separate one applies here as well.